MembershipApp/templates/MembershipApp/about.html

```diff
@@ -0,0 +1,134 @@
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+{% endblock %}
+<footer class="space--sm footer-2">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6 col-md-3 col-xs-6">
+                <h6 class="type--uppercase">Company</h6>
+                <ul class="list--hover">
+                    <li><a href="#">About Company</a></li>
+                    <li><a href="#">Our Team</a></li>
+                    <li><a href="#">Locations</a></li>
+                    <li><a href="#">History</a></li>
+                    <li><a href="#">Work With Us</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-6 col-md-3 col-xs-6">
+                <h6 class="type--uppercase">Apply</h6>
+                <ul class="list--hover">
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Apply to camp</a></li>
+                    <li><a href="#">Apply to speak</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-6 col-md-3 col-xs-6">
+                <h6 class="type--uppercase">BEE</h6>
+                <ul class="list--hover">
+                    <li><a href="#">Burning Man 2019</a></li>
+                    <li><a href="#">Community Ethos</a></li>
+                </ul>
+            </div>
+            <div class="col-sm-6 col-md-3 col-xs-6">
+                <h6 class="type--uppercase">Support</h6>
+                <ul class="list--hover">
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#">FAQ</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-6"><span class="type--fine-print">© <span class="update-year">2020</span> BubblyEverAfter</span>
+                <a class="type--fine-print" href="#">Privacy Policy</a> <a class="type--fine-print" href="#">Legal</a>
+            </div>
+            <div class="col-sm-6 text-right text-left-xs">
+                <ul class="social-list list-inline list--hover">
+                    <li><a href="#"><i class="socicon socicon-google icon icon--xs"></i></a></li>
+                    <li><a href="#"><i class="socicon socicon-twitter icon icon--xs"></i></a></li>
+                    <li><a href="#"><i class="socicon socicon-facebook icon icon--xs"></i></a></li>
+                    <li><a href="#"><i class="socicon socicon-instagram icon icon--xs"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</footer>
+</div>
+<div class="main-container"></div>
+<script src="{% static "MembershipApp/js/jquery-3.1.1.min.js" %}"></script>
+<script src="{% static "MembershipApp/js/parallax.js" %}"></script>
+<script src="{% static "MembershipApp/js/smooth-scroll.min.js" %}"></script>
+<script src="{% static "MembershipApp/js/scripts.js" %}"></script>
```

Comment:
you don't need these, please move them to script page if they are being used by all the templates

Reply:
apply this to all templates